score/templates/secrets.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "score.fullname" . }}
+  labels:
+    app: {{ include "score.name" . }}
+    chart: {{ include "score.chart" . }}
+    release: {{ .Release.Name | quote }}
+    heritage: {{ .Release.Service | quote }}
+type: Opaque
+data:
+  s3-secretKey: {{ .Values.appConfig.objectStorage.secretKey }}
+  auth-clientSecret: {{ .Values.appConfig.authServer.clientSecret }}
```

Comment:
why not generate this secret here instead of passing it in values, you can keep both options